posthog/api/test/__snapshots__/test_decide.ambr

```diff
@@ -666,7 +668,9 @@
 ---
 # name: TestDecide.test_flag_with_regular_cohorts.8
   '
-  SELECT ("posthog_person"."properties" -> '$some_prop_1') = '"something_1"' AS "flag_X_condition_0"
+  SELECT (("posthog_person"."properties" -> '$some_prop_1') = '"something_1"'
+          AND "posthog_person"."properties" ? '$some_prop_1'
+          AND NOT (("posthog_person"."properties" -> '$some_prop_1') = 'null')) AS "flag_X_condition_0"
```

Comment:
These two do look like unrelated changes. They're definitely referencing SQL that's not generated by HogQL. I remember hearing about some changes to `/decide`, so that's probably this plus a bad merge somewhere by someone?

Reply:
Yeah me 🙈 , old PR got merged after a snapshot change went in, which didn't update snapshots again